palette/src/rgb/rgb.rs

```diff
@@ -944,11 +946,6 @@ impl From<ParseIntError> for FromHexError {
     }
 }
 
-impl From<&'static str> for FromHexError {
-    fn from(err: &'static str) -> FromHexError {
-        FromHexError::HexFormatError(err)
-    }
-}
```

Comment:
Sorry if I wasn't clear before. We need to keep this for now, so it's backwards compatible, but it doesn't have to change to produce any RGBA errors. I can remove it later, for 0.8.0.

Reply:
It's technically still breaking, depending on how you see it, since it changes the output for anything that contains "rgba" somewhere. In practice, I doubt anyone is relying on it at all.

I think the simplest option is to leave it unchanged and stop using it, or at least not use it for new errors. Then it still behaves exactly as before. I can make an issue to clean it up further.

Reply:
You can add a documentation comment to the impl block that explains what it outputs if you want. 🙂

Reply:
Great, thank you! 👍